packages/protobuf-conformance/failing_tests_without_bigint.txt

```diff
@@ -0,0 +1,30 @@
+# proto2 extensions are not implemented
+# WARNING, test=Recommended.Proto2.JsonInput.FieldNameExtension.Validator: Expected JSON payload but got type 1. request=json_payload: "{\n        \"[protobuf_test_messages.proto2.extension_int32]\": 1\n      }" requested_output_format: JSON message_type: "protobuf_test_messages.proto2.TestAllTypesProto2" test_category: JSON_TEST, response=parse_error: "Error: cannot decode message protobuf_test_messages.proto2.TestAllTypesProto2 from JSON: key \"[protobuf_test_messages.proto2.extension_int32]\" is unknown"
+Recommended.Proto2.JsonInput.FieldNameExtension.Validator
+
+# Without BigInt support, a JavaScript runtime can only safely represent values
+# in the range from Number.MIN_SAFE_INTEGER to Number.MAX_SAFE_INTEGER.
+# That means number values in JSON will be unsafe as well. If you plan to use
+# protobuf-es on runtimes without BigInt support, make sure that the JSON data
+# you consume does contains 64-bit values as strings instead of numbers.
```

Comment:
It's worth noting that the [JSON mapping](https://developers.google.com/protocol-buffers/docs/proto3#json) specifies a string for all 64-bit integrals. It also specifies that parsers should be lenient and accept numbers too, but we cannot reasonable conform to that without a bespoke JSON parser.

This is not an issue with BigInt support, because we convert numbers to BigInt right away, and modern runtimes don't lose precision on that path.